src/object/object_domain.c

```diff
@@ -9606,6 +9607,76 @@ tp_value_cast_internal (const DB_VALUE * src, DB_VALUE * dest, const TP_DOMAIN *
 	  }
 	  break;
 
+
+	case DB_TYPE_VECTOR:
+	  {
+	    DB_VALUE element;
+	    SETREF *setref = db_get_set (src);
+	    if (setref)
+	      {
+		DB_VALUE str;
+		db_value_domain_init (&str, DB_TYPE_VARCHAR, TP_FLOATING_PRECISION_VALUE, 0);
+
+		std::ostringstream oss;
+
+		oss << "[";
+
+		int cardinality = db_set_size (setref);
+		for (int i = 0; i < cardinality; i++)
+		  {
+		    if (db_set_get (setref, i, &element) != NO_ERROR)
+		      {
+			status = DOMAIN_ERROR;
+			break;
+		      }
+
+		    tp_ftoa (&element, &str);
+
+		    if (DB_IS_NULL (&str))
+		      {
+			status = DOMAIN_ERROR;
+			break;
+		      }
+
+		    oss << db_get_string (&str);
+		    if (i < cardinality - 1)
+		      {
+			oss << ",";
+		      }
+		    db_value_clear (&str);
+		  }
+		oss << "]";
+
+		char *new_string = db_private_strdup (NULL, oss.str ().c_str ());
+		if (!new_string)
+		  {
+		    status = DOMAIN_ERROR;
+		    break;
+		  }
+
+		int new_string_len = oss.str ().size ();
```

Comment:
```suggestion
                // check varchar's precision e.g.) 'select cast (vec as varchar (1)) from tbl;'
		int new_string_len = oss.str ().size ();
```

이후 코드 파악을 위해 주석이 한줄 있으면 좋을 것 같습니다